loops_lab.js

```diff
@@ -0,0 +1,137 @@
+/*
+1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
+*/
+
+let num = 15; // for #1 & #2, I wanted to combine both while and for loops, and not do them seperate. was not sure which way you wanted, but I did it.
+while ( num >= 1){
```

Comment:
There is no need for two loops in this case as looping through one set of number (0, 1, 2, 3, .... 15) is enough for this question.